stanford/roles/cluster/defaults/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+CLUSTER_AMI: ami-73f7da13  # 16.04
```

Comment:
Should we just remove this for now?
I defaulted to leaving, but overriding it, just in case.